routes/getDataForQeues.js

```diff
@@ -70,7 +70,7 @@ module.exports = async function getDataForQeues({ queues, query = {} }) {
       return {
         name,
         counts,
-        jobs: jobs.map(formatJob),
+        jobs: jobs.filter((job) => !!job).map(formatJob),
```

Comment:
A `reduce` instead of a `filter` -> `map` would reduce the iteration count by up to 2x 🙂 

I'd recommend going with that instead for this case!

Reply:
return {
      name,
      counts,
      jobs: jobs
        .reduce((result, job) => {
          if (job) {
            result.push(formatJob(job))
          }
          return result
        }, [])
    }

Reply:
I don't agree with you, this consumes 2x memory, it is better to use `filter` & `map` as it is more declarative

Reply:
I don't agree with you, this consumes 2x memory, it is better to use filter & map as it is more declarative